### PR TITLE
[AMORO-2338] Incorrect display of props in hive upgrade page

### LINKED ISSUE #2338: [Bug]: Incorrect display of props in hive upgrade page
### What happened?

image
As you can see in the figure, I get props that are not as expected when I perform a hive upgrade operation

### Affects Versions

0.6.x

### What engines are you seeing the problem on?

AMS

### How to reproduce

1. Upgrade a hive table to mixed-hive table
2. Click the properties tab

### Relevant log output

_No response_

### Anything else

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by shendanfengg**
## Why are the changes needed?

Close #2338 .

## Brief change log

-Modify the filter for props

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wangtaohz** [comment]
The title of the PR is supposed to be `[AMORO-XXXX]` @shendanfengg

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java [hunk 1]
```diff
@@ -243,6 +243,7 @@ public void getUpgradeHiveTableProperties(Context ctx) throws IllegalAccessExcep
         AmsUtil.getNotDeprecatedAndNotInternalStaticFields(TableProperties.class);
     tableProperties.keySet().stream()
         .filter(key -> !key.endsWith("_DEFAULT"))
+        .filter(key -> tableProperties.containsKey(key + "_DEFAULT"))
```
Comment (wangtaohz) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java:246]: If I understand correctly, properties without default values will be ignored. I'm curious to know which specific parameters need to be ignored in this way. Would it be appropriate to consider handling these parameters through HIDDEN_EXPOSED?

Comment (shendanfengg) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java:246]: By my count there are close to 30 props that need to be filtered.

**Comment by shendanfengg** [comment]
> The title of the PR is supposed to be `[AMORO-XXXX]` @shendanfengg
I've modified it.

**Comment by czy006** [comment]
@shendanfengg @wangtaohz I think we should continue to improve this PR and rebase it.I will try to continue

**Pull request closed without merging**